compose.langflow.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  langflow:
+    container_name: ${HARBOR_CONTAINER_PREFIX:-harbor}.langflow
+    image: langflowai/langflow:${HARBOR_LANGFLOW_VERSION:-latest}
+    ports:
+      - "${HARBOR_LANGFLOW_HOST_PORT:-7860}:7860"
+    volumes:
+      - ${HARBOR_LANGFLOW_DATA:-/data/harbor/langflow}:/var/lib/langflow
+      - ./langflow/configs:/app/configs
+    env_file:
+      - ./.env
+      - ./langflow/override.env
+    environment:
```

Comment:
Everything in the override.env can be removed here - will make it easier to adjust via `harbor env`